target_hubspot_v4/unified.py

```diff
@@ -205,11 +205,10 @@ def process_contacts(self, record):
             row.update({"id": record.get("id")})
 
         if "id" not in row and row["properties"].get("email"):
-            contact_search = search(dict(self.config), row["properties"].get("email"))
+            contact_search = search_contact_by_email(dict(self.config), row["properties"].get("email"))
             if contact_search:
-                if len(contact_search) > 0:
-                    if contact_search[0].get("id"):
-                        row.update({"id": contact_search[0]["id"]})
+                if contact_search.get("id"):
```

Comment:
Can't we make the `if` above this
```py
if contact_search and  contact_search.get("id"):
```
any reason to have it nested?

Reply:
My negligence, mixed up two APIs logic, search and direct object by email.